xpk.py

```diff
@@ -1127,6 +1142,120 @@ def zone_to_region(zone) -> str:
+    return False
+  return True
+
+def create_service_account(args) -> int:
+  """Creates a service account in the project.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  command = (
+    f'gcloud iam service-accounts create {args.project}-{_XPK_SERVICE_ACCOUNT} \
+    --description="Service Account for XPK" \
+    --display-name="{args.project}-{_XPK_SERVICE_ACCOUNT}"'
+  )
+  return_code = run_command_with_updates(command, 'Service Account Create', args)
+  if return_code != 0:
+    xpk_print(f'Service Account Create request returned ERROR {return_code}')
+    xpk_print('Make sure you have Service Account Admin Role attached to your user.')
+    return 1
+  return 0
+
+def get_existing_roles_in_service_account(args) -> set:
+  """
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    set of IAM roles already attached to the service account.
+  """
+  roles = []
+  service_account_name = get_service_account_name(args)
+  command = (
+    f'gcloud projects get-iam-policy {args.project}'
+    f' --filter="bindings.members:{service_account_name}"'
+    ' --flatten="bindings[].members" --format="table(bindings.role)"'
+  )
+  return_code, return_value = run_command_for_value(command, 'Get IAM Roles For Service Account', args)
+  if return_code != 0:
+    xpk_print(f'Get IAM Roles For Service Account request returned ERROR {return_code}')
+  else:
+    return_value = return_value.strip()
+    roles = set(return_value.split('\n'))
+    """Format of return_value is:
+          ROLE
+          roles/storage.admin
+          roles/logging.admin
+      removing `ROLE` from the list
+    """
+    if 'ROLE' in roles:
+      roles.remove('ROLE')
+  return roles
+
+
+def add_roles_to_service_account(args) -> int:
+  """Add IAM roles to service account.
+
+  Args:
+    args: user provided arguments for running the command.
+  
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  service_account_name = get_service_account_name(args)
+  existing_roles = get_existing_roles_in_service_account(args)
+  xpk_print(f'IAM roles already attached to service account: {existing_roles}')
+
+  for name, role in IAMRoles.items():
+    if role in existing_roles:
+      continue
+
+    xpk_print(f'Adding {name} role to service account: {service_account_name}.')
+    command = (
+      f'gcloud projects add-iam-policy-binding {args.project} \
+      --member="serviceAccount:{service_account_name}" \
+      --role="{role}" \
+      --condition=None'
+    )
+    return_code = run_command_with_updates(command, 'Add IAM Role to Service Account', args, verbose=False)
+    if return_code != 0:
+      xpk_print(f'Add IAM Role to Service Account request returned ERROR {return_code}')
+      xpk_print('Make sure you have Project IAM Admin Role attached to your user.')
```

Comment:
Can you add a print statement that says: xpk_print('Please look at https://github.com/google/xpk/blob/main/README.md#permission-issues-requires-one-of-permission_name-permissions for how to add permissions to your user.')